Binary files mod_mono-1.1.10/.mod_mono.conf.in.swp and mod_mono-1.1.10-new/.mod_mono.conf.in.swp differ
diff -uNr mod_mono-1.1.10/mod_mono.conf.in mod_mono-1.1.10-new/mod_mono.conf.in
--- mod_mono-1.1.10/mod_mono.conf.in	2005-11-02 13:55:11.000000000 -0500
+++ mod_mono-1.1.10-new/mod_mono.conf.in	2005-11-28 20:12:04.000000000 -0500
@@ -21,5 +21,14 @@
     DirectoryIndex index.aspx
     DirectoryIndex Default.aspx
     DirectoryIndex default.aspx
-</IfModule>
 
+    Alias /mono-test "@FINKPREFIX@/share/doc/mono-xsp/test"
+    MonoExecutablePath "@FINKPREFIX@/bin/mono"
+    MonoServerPath "@FINKPREFIX@/lib/xsp/1.0/mod-mono-server.exe"
+    MonoRunXSP True
+
+    <Directory @FINKPREFIX@/share/doc/mono-xsp/test>
+        SetHandler mono
+    </Directory>
+
+</IfModule>
